package.json

```diff
@@ -5,6 +5,13 @@
   "main": "./lib/index.js",
   "types": "./lib/index.d.ts",
   "type": "commonjs",
+  "browser": {
+    "./lib/index.js": "./lib/index.js",
+    "./lib/hooks/useState":"./lib/hooks/useState.js",
```

Comment:
Guess we should add an index file inside /hooks to export `useState` and potential future hooks.

Reply:
Yes, instead of manually exposing every hook. That would be great, i will add that